mojaloop/values.yaml

```diff
@@ -13224,9 +13224,9 @@ ml-testing-toolkit:
       ## We can pass the JSON content as the value for the parameters
       ## Or we can pass a http/https URL for the JSON file as the value for the parameters. Then the file will be downloaded and replaced in the corresponding location.
       ## Ex: rules_callback__default.json: "https://raw.githubusercontent.com/mojaloop/ml-testing-toolkit/master/spec_files/rules_callback/default.json"
-      rules_callback__default.json: "https://github.com/mojaloop/testing-toolkit-test-cases/raw/v15.3.0-snapshot/rules/mojaloop/ml-testing-toolkit/spec_files/rules_callback/default.json"
-      rules_response__default.json: "https://github.com/mojaloop/testing-toolkit-test-cases/raw/v15.3.0-snapshot/rules/mojaloop/ml-testing-toolkit/spec_files/rules_response/default.json"
-      rules_validation__default.json: "https://github.com/mojaloop/testing-toolkit-test-cases/raw/v15.3.0-snapshot/rules/mojaloop/ml-testing-toolkit/spec_files/rules_validation/default.json"
+      rules_callback__default.json: "https://github.com/mojaloop/testing-toolkit-test-cases/raw/v15.2.0/rules/mojaloop/ml-testing-toolkit/spec_files/rules_callback/default.json"
```

Comment:
Do we need to leave `v15.3.0-snapshot` version for _testing-toolkit-test-cases_ ?

Reply:
Yes, it will be upgraded during release. `15.3.0-snapshot` has some changes for the next release.